lib/time.py

```diff
@@ -23,3 +62,67 @@ def sleep(secs):
 
 def time():
   return float(Now().UnixNano()) / Second
+
+
+_strftime_directive_map = {
+    '%': '%',
+    '0': NotImplementedError,
```

Comment:
Linux output following by the way, 
```
>>> time.strftime('%10')
'       %10'
```
if I remove from that list, output of `'%10'` will be 
```
'%10'
```